pom.xml

```diff
@@ -17,16 +17,13 @@
 		<!-- DEPENDENCIES VERSION -->
 		<jdk.version>17</jdk.version>
 		<cds.services.version>2.1.0</cds.services.version>
+		<xsuaa.version>3.0.1</xsuaa.version>
```

Comment:
Maybe we should rather use the `cds-starter-cloudfoundry` in srv/pom.xml which also includes all required XSUAA dependencies with an exclusion for the cds-feature-hana and cds-feature-mt? What do you think? This would allow us to use the XSUAA version coming from CAP.

Reply:
Ah, dumb me. Thanks!